robottelo/cli/hostgroup.py

```diff
@@ -30,6 +30,30 @@ class HostGroup(Base):
 
     command_base = 'hostgroup'
 
+    @classmethod
+    def ansible_roles_assign(cls, options):
+        """Assigns the associated ansible-roles"""
+        cls.command_sub = 'ansible-roles assign'
+        return cls.execute(cls._construct_command(options), output_format='csv')
+
+    @classmethod
+    def ansible_roles_remove(cls, options=None):
+        """Remove ansible roles"""
+        cls.command_sub = 'ansible-roles remove'
+        return cls.execute(cls._construct_command(options), output_format='csv')
+
+    @classmethod
+    def ansible_roles_add(cls, options):
+        """Assigns the associated ansible-roles"""
+        cls.command_sub = 'ansible-roles add'
+        return cls.execute(cls._construct_command(options), output_format='csv')
```

Comment:
Based off of hammer and the code below I can't tell the difference between `hammer hostgroup ansible-roles add` vs `hammer hostgroup ansible-roles assign`. Below it looks like assign can do multiple roles and add only does one? If there is a difference can you add it to the docstrings here, and if not then let's not worry about it too much.

Reply:
Yeah, you are right about the functionality I'll update the docstring.